openest/dmas/remote.py

```diff
@@ -4,7 +4,8 @@ def view_model(objtype, id):
       print("Accessing url...")
       try:
         return StoredModel.create_from_url(id, (float('-inf'), float('inf')))
-      except:
+    except Exception as ex:  # CATBELL
```

Comment:
Looks like your indentation is wrong here.

Reply:
Good catch. I missed that. Thank you! Resolved with d52c86f7ab63bdf409c3fef3829c0c244f10eab6